src/transformers/models/pegasus/modeling_flax_pegasus.py

```diff
@@ -223,7 +223,7 @@ def shift_tokens_right(input_ids: np.array, pad_token_id: int, decoder_start_tok
 
 
 # Copied from transformers.models.marian.modeling_flax_marian.create_sinusoidal_positions
-def create_sinusoidal_positions(n_pos, dim, dtype):
```

Comment:
This function is copied from and does not do anything with `dtype` @sanchit-gandhi so adapted its two use below in the file.